spec/models/translation_override_spec.rb

```diff
@@ -47,6 +47,23 @@
     expect(ovr.value).to eq('some value')
   end
 
+  it "doesn't create records for new values if they match original values" do
+    I18n.backend.store_translations(:en, keyfortests: "some text")
+
+    TranslationOverride.upsert!(:en, 'keyfortests', 'some text')
+    expect(TranslationOverride.where(translation_key: 'keyfortests').count).to eq(0)
+  end
+
+  it "destroys existing record if new value matches original" do
+    I18n.backend.store_translations(:en, keyfortests: "some text")
+
+    TranslationOverride.upsert!(:en, 'keyfortests', 'some text 2', bypass_reload: true)
```

Comment:
Is the `bypass_reload` parameter only used by this test? Could you try rewriting the test so that this parameter isn't needed?

Reply:
@gschlager I could use the key of a real translation for this test. Does that sound good to you?

Reply:
Yeah reusing a translation is fine here, it will be clear how to fix it if needed.

Reply:
OK, done 👍